src/main/scala/io/sdkman/changelogs/JoernMigrations.scala

```diff
@@ -0,0 +1,26 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "071")
+class JoernMigrations {
+
+  @ChangeSet(order = "001", id = "001_add_joern_1_1_524", author = "ursachec")
+  def migration001(implicit db: MongoDatabase) = {
+    Candidate(
+      candidate = "io.joern",
+      name = "joern-cli",
```

Comment:
This is the pretty print name that we use on our website and in our serverside templates. `Joern` is probably best here.